### PR TITLE
content/projects/_bldrs.md: rename from _blrds and fill in initial content.

### PR DESCRIPTION
**Pull request by pablo-mayrgundter**
Hi Max,

Here's our project page submission.

Thanks!
Pablo

### CODE REVIEW

**Review by n0rdlicht — CHANGES_REQUESTED** [review]
Hi @pablo-mayrgundter many thanks for submitting this. For the project to show up in the preview (just approved the preview build) you'll unfortunately need rename the file, see inline comment.

cc @MaximilianVomhof

File: content/projects/_bldrs.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+Bldrs is a team of builders based in the USA, Australia and Switzerland with a
+centurty of combined experience in CAD, IT and Construction.
+
+
+### Contact
+
+Let's Build Every Thing Together!
+- [Bldrs Discord](https://discord.gg/9SxguBkFfQ)
+- [Bldrs Share GitHub](https://github.com/bldrs-ai/Share)
```
[the remark this answers is not in the document]
Comment (pablo-mayrgundter) [reply content/projects/_bldrs.md]: Tried just putting them all up top

File: content/projects/_bldrs.md
[the remark this answers is not in the document]
Comment (pablo-mayrgundter) [reply content/projects/_bldrs.md:1]: Done

**Comment by pablo-mayrgundter** [comment]
PTAL

**Comment by n0rdlicht** [comment]
nice - lgtm, looks good to me. @MaximilianVomhof please check https://opensource-construction-gs7jjvk0o-opensource-construction.vercel.app/projects/bldrs/ before we merge.

**Review by n0rdlicht — APPROVED** [review]

**Pull request merged**